### ISSUE TITLE
Too many books in bookshelves == chunk reset ? :(

### ISSUE THREAD

**Issue by ghost** [Nuchaz/BiblioCraft#349]
Hi !
TL;DR:
- I play on Bibliocraft 1.11.4
- Modpack is infinity 2.4.2
- Minecraft version is 1.7.10
- When I put 3.2mB of text in a single chunk in the form of printed books in bibliocraft oak bookshelves the chunk rollbacks when the server restarts
- I tried the same setup with diamond chests this time and it worked perfectly

Long version:

I'm doing a cool project. Basically, I'm building a huge library of the worst litterature ever written, namely, the lowest ranked Minecraft fanfics over on fanfictions.net... in ingame books format !

I'm telling you that because you may want the numbers. I have 456 bookshelves equally spread accross 6 chunks (76 each). Unfortunately, it seems like ~30 of them are already enough to cause the chunk to rollback when the server restarts. 

The whole fanfic deal weights 22 mB, the chunk contained about 3.2 megabytes of text each time when before it rolled back (I know which .txt files they contained). 

Also, a book is 16 pages long, each page has 21 lines which contain max 25 chars each. Each bookshelf contains 16 books.
27 bookshelves were fully stocked, while 30 remained empty in that chunk. I know which fanfics were in those books and they contained 3.2mB of text combined.

I tried thrice, every time I log off and back, the first cubicle of bookshelves gets rolled back (the only one that's 100% filled up)

Here's the room:
http://imgur.com/a/cfhxG

I tried the same setup with diamond chests this time and it worked perfectly

I play singleplayer, open to lan sometimes

**Comment by Nuchaz** [comment]
I had never experienced this issue. That is really unfortunate. I love what you were working on. Sounds awesome. Sorry it took so long to respond. Hopefully this doesn't happen in newer versions. Enough code was rewritten in the newer versions, theres a good chance this was solved.
